src/components/transactions/TxSummary/index.tsx

```diff
@@ -75,7 +75,7 @@ const TxSummary = ({ item, isGrouped }: TxSummaryProps): ReactElement => {
         </Box>
       )}
 
-      <Box gridArea="status">{isQueue && !isPending ? <QueueActions tx={tx} /> : <TxStatusLabel tx={tx} />}</Box>
+      <Box gridArea="status">{isQueue && !isPending ? <QueueActions tx={tx} /> : <PendingActions tx={tx} />}</Box>
```

Comment:
I think it is a bit misleading that we are rendering `PendingActions` for transactions that are in the history because the `TxStatusLabel` was moved in there. It would be more clear to keep the separation in this component i.e.

```
<Box gridArea="status">
    {isQueue ? isPending ? <PendingActions tx={tx} /> : <QueueActions tx={tx} /> : <TxStatusLabel tx={tx} />}
</Box>
```

Reply:
Good point. I'll try to come up with something. `?? ::` too complicated :D

Reply:
Now should be better. I realized that queue and pending are basically the same. So moved the pending stuff to the queue (you can't have pending in the history tab). Also added a new grid cell for actions. Now the UI no longer jumps

Reply:
Thanks!